app/components/TagChip/TagChip.tsx

```diff
@@ -0,0 +1,13 @@
+import { FunctionComponent } from "react";
+
+type TProps = {
+  children: string;
+};
+
+export const TagChip: FunctionComponent<TProps> = ({ children }) => {
```

Comment:
Just use `chip`, no need for Tag and Chip in a single component, it's basically the same

Reply:
Done